include/exiv2/xmpsidecar.hpp

```diff
@@ -83,6 +84,6 @@ namespace Exiv2 {
     //! Check if the file iIo is an XMP sidecar file.
     EXIV2API bool isXmpType(BasicIo& iIo, bool advance);
 
-}                                       // namespace Exiv2
+}  // namespace Exiv2
 
-#endif                                  // #ifndef XMPSIDECAR_HPP_
+#endif  // #ifndef XMPSIDECAR_HPP_
```

Comment:
These ending comments I find interesting. In other codebases that I've seen they just have the define name as a comment. I think google's cpplint also complains about it.

Reply:
I am not sure what is your point here. clang-format is just reformatting here the spaces between the `#endif` and the comment.